engine/execution/ingestion/engine.go

```diff
@@ -717,6 +719,14 @@ func (e *Engine) executeBlock(
 
 }
 
+func transactionCount(result flow.ExecutionResult) uint64 {
```

Comment:
Will this include the system transaction?

Reply:
it does not, so this only report the user transaction (non-system transactions). system transaction is always a fixed single transaction, so I don't think we need to +1.

Reply:
should it be called `nonSystemTransactionCount` then ?